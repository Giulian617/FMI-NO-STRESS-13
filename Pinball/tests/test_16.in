100 5000
46 53 5
15 46 7
32 53 6
42 53 1
93 53 7
69 46 1
39 32 4
45 46 7
83 93 6
22 93 5
67 53 5
95 45 7
26 69 8
19 69 10
14 95 1
68 45 4
64 15 6
8 68 8
40 67 8
63 39 3
82 14 5
94 8 9
35 32 3
49 40 3
79 53 3
13 32 5
99 67 8
12 39 7
65 79 5
3 79 10
37 82 5
73 8 2
58 67 10
98 14 6
74 69 7
54 73 8
91 14 4
80 91 9
52 49 7
75 26 6
36 39 2
5 42 5
100 8 7
90 73 6
97 19 8
92 45 2
30 3 3
33 65 3
23 69 6
88 100 10
29 82 9
16 58 1
59 29 10
55 74 8
34 73 6
2 46 7
81 45 6
72 98 2
7 13 1
66 72 4
17 58 4
28 82 8
62 19 1
9 13 1
51 83 4
31 9 8
38 72 5
47 42 9
6 82 6
43 74 4
25 19 2
27 3 8
50 28 2
84 98 10
4 12 4
57 31 4
10 5 1
44 29 2
76 7 8
60 39 2
85 80 10
61 17 2
1 65 4
78 33 8
70 7 1
96 95 8
86 59 8
20 6 7
89 36 9
11 14 1
21 3 4
24 25 7
48 40 7
41 5 5
77 55 1
71 43 7
56 30 9
18 94 1
87 61 6
