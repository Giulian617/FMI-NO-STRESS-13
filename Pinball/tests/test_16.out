551047625
341117606
720629263
671291085
366311714
972885664
995790652
62116106
495160617
672068366
881933270
460317880
713310080
246928344
383344578
505647060
848040134
895418071
756683824
979210059
33570029
737031944
439365801
265030459
965521282
923021331
263581324
205412866
282173836
683367
341117606
11018431
742439736
46011292
497047282
862314684
950676785
892765469
89234922
457454353
80965880
416834764
159768672
299580804
205412866
246928344
656727825
937604620
120078566
351373038
